src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -19,6 +19,8 @@ public class LibGpiodDriver : UnixDriver
 
         protected internal override int PinCount => Interop.libgpiod.gpiod_chip_num_lines(_chip);
 
+        private string consumerName = System.Diagnostics.Process.GetCurrentProcess().ProcessName;
```

Comment:
- this can be static
- naming convention for static is: `s_consumerName` for instance fields: `_consumerName`

Reply:
Thanks for the review @krwq